pettingzoo/utils/wrappers/base.py

```diff
@@ -35,18 +40,20 @@ def __init__(self, env):
 
         # Not every environment has the .state_space attribute implemented
         try:
-            self.state_space = self.env.state_space
+            self.state_space = (
+                self.env.state_space  # pyright: ignore[reportGeneralTypeIssues]
+            )
         except AttributeError:
             pass
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str):
```

Comment:
I think this is fine because double underscore private methods don’t generally have return types

Reply:
Actually nvm later on you use -> Any for getattr, might as well to be honest, anything besides init